### ISSUE TITLE
Component: Bride Review

### ISSUE THREAD

**Issue by erinkelley27** [jserrao/modern-trousseau-gatsby#99]
**Bride Review Component**

Desktop:
<img width="681" alt="Screen Shot 2020-02-04 at 9 48 55 PM" src="https://user-images.githubusercontent.com/38398204/73806702-38193a80-4798-11ea-9b90-526c5f6206e0.png">

Mobile:
<img width="371" alt="Screen Shot 2020-02-04 at 9 48 43 PM" src="https://user-images.githubusercontent.com/38398204/73806710-3bacc180-4798-11ea-9003-56567ca16657.png">

**Comment by jserrao** [comment]
@jmripper - I think this is what you are working on...

**Comment by jmripper** [comment]
@jserrao yeah this is about 90% complete. just need to fix the images and quotes used.
